### PR TITLE
fixed a few typos and improved clarity for systemd

### PR DESCRIPTION
**Pull request by daavve**
Hi.  I notices a few small issues with the Systemd documentation.  Please let me know if it needs any other fixes.

-Dave

### CODE REVIEW

File: docs/source/deploy.rst [hunk 1]
```diff
@@ -251,27 +251,22 @@ unix socket:
 
     d /run/gunicorn 0755 someuser somegroup -
 
-Next enable the services so they autostart at boot::
+Next enable the socket so they it autostarts at boot::
```
Comment (berkerpeksag) [inline docs/source/deploy.rst]: Do we still need "they" or can we remove it now?

File: docs/source/deploy.rst [hunk 2]
```diff
@@ -251,27 +251,22 @@ unix socket:
...
 Either reboot, or start the services manually::
 
-    systemctl start gunicorn.service
     systemctl start gunicorn.socket
 
 
-After running ``curl --unix-socket /run/gunicorn/socket http:``, Gunicorn
-should start and you should see something like that in logs::
+After running ``curl --unix-socket /run/gunicorn/socket http``, Gunicorn
+should start and you should see some html from your server.
```
Comment (berkerpeksag) [inline docs/source/deploy.rst]: html -> HTML

File: docs/source/deploy.rst [hunk 3]
```diff
@@ -251,27 +251,22 @@ unix socket:
...
 
-    2013-02-19 23:48:19 [31436] [DEBUG] Socket activation sockets:
-    unix:/run/gunicorn/socket,http://0.0.0.0:9000,http://[::]:8000
+You must now configure your web proxy to send traffic to the new Gunicorn
+socket. Configure your nginx.conf like the following:
```
Comment (berkerpeksag) [inline docs/source/deploy.rst]: \`\`nginx.conf\`\`

**Comment by daavve** [comment]
Thank you so much for pointing you my mistakes!  Let me know if you find anything else.

**Comment by berkerpeksag** [comment]
Looks great, thanks! :)

**Pull request merged**